tests/integration-tests-topologies/src/main/java/org/apache/pulsar/tests/topologies/PulsarCluster.java

```diff
@@ -72,26 +83,45 @@ public static PulsarCluster forSpec(PulsarClusterSpec spec) {
     private final ProxyContainer proxyContainer;
 
     private PulsarCluster(PulsarClusterSpec spec) {
+
         this.spec = spec;
         this.clusterName = spec.clusterName();
         this.network = Network.newNetwork();
+
         this.zkContainer = new ZKContainer(clusterName)
+            .withLogConsumer(new Slf4jLogConsumer(log).withPrefix(ZKContainer.NAME))
             .withNetwork(network)
             .withNetworkAliases(ZKContainer.NAME)
+            .withCreateContainerCmdModifier(createContainerCmd -> {
```

Comment:
pulsar container already set the host name and name, why do you need to change here?

https://github.com/apache/incubator-pulsar/blob/master/tests/integration-tests-topologies/src/main/java/org/apache/pulsar/tests/containers/PulsarContainer.java#L84

for common changes, it should go to the base class, rather than spreading all over the places.

Reply:
will remove it.